src/components/Internationalize.js

```diff
@@ -5,20 +5,15 @@
  */
 
 import React from 'react';
-import { IntlProvider } from 'react-intl';
-import i18n from '../utils/i18n';
+import { RawIntlProvider } from 'react-intl';
 
 type Props = {
     children?: any,
     intl: Object,
 };
 
 const Internationalize = ({ children, intl }: Props) => {
```

Comment:
Might be worth trashing this component at this point. It's just a proxy for RawIntlProvider now.